java/memory/src/main/java/io/netty/buffer/ArrowBuf.java

```diff
@@ -1204,4 +1204,45 @@ public ArrowBuf reallocIfNeeded(final int size) {
               "Realloc is only available in the context of operator's UDFs");
     }
   }
+
+  /**
+   * Following are wrapper methods to keep this backward compatible.
+   */
+  public void release() {
```

Comment:
can we mark these as "@deprecated", so clients know they should moving off of them?

Reply:
done.

Reply:
why deprecate ? It's a useful convenience fn.